packages/types/src/index.ts

```diff
@@ -1,20 +1,50 @@
-export { DatabaseInterface as DBInterface } from './types/database-interface';
-import { ConnectionInformations as ConnectionInformationsType } from './types/connection-informations';
+import { DatabaseInterface } from './types/database-interface';
```

Comment:
You can express importing and exporting like this. 

```javascript
`export * from './types/database-interface`
```

This will export all named export within `./types/database-interface` and you no longer need to explicitly import.

Reply:
I did this at the moment so i can rename the interfaces on the fly, but I guess it can be written with stars in #197